services-camel/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -144,7 +144,7 @@
     </bean>
     
     <bean id="binaryMetadataProcessor" class="edu.unc.lib.dl.services.camel.BinaryMetadataProcessor">
-        <constructor-arg value="${fcrepo.binaryBase}"/>
+        <property name="baseBinaryPath" value="${fcrepo.binaryBase}" />
```

Comment:
just wondering what your motivation was for changing the way the med processor gets initialized. i noticed above that you removed the constructor in favor of a setter

Reply:
I switched it mainly because I needed to be able to programmatically provide the baseBinaryPath with a temp directory in a test, but it also seems more in line with typical dependency injection patterns.